cogs/resourcecog.py

```diff
@@ -0,0 +1,169 @@
+    rinfo = cfg[str(msg.id)]
+    rscembed = discord.Embed(
+        title=rinfo["name"], description=rinfo["desc"], color=0x66FF99
+    )
+    rscembed.add_field(name="Link", value=rinfo["link"], inline=True)
+    rscembed.add_field(name="Tags", value=rinfo["tags"], inline=True)
+    rscembed.set_footer(text="Submitted by " + rinfo["msgauth"])
+    await rsc_channel.send(embed=rscembed)
+    del cfg[str(msg.id)]
+    jsonsave(cfg)
+
+
+class ResourceCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+        with open("config.json", "r") as f:
+            cfg = json.load(f)
+            try:
+                self.resource_channel_id = cfg["resourceChannel"]
+            except KeyError:
+                self.resource_channel_id = None
+                print("Remember to !setresourcechannel")
+            try:
+                self.review_channel_id = cfg["reviewChannel"]
+            except KeyError:
+                self.review_channel_id = None
+                print("Remember to !setreviewchannel")
+
+    @commands.command()
+    @commands.has_role("Bot Commander")
+    async def setresourcechannel(self, ctx, channel_id):
+        """Enter ID of channel to set as resource channel"""
+        rsc_channel = await self.bot.fetch_channel(channel_id)
+
+        if rsc_channel is None:
+            await ctx.send("Invalid channel id")
+            return
+
+        with open("config.json", "r") as f:
+            cfg = json.load(f)
+            cfg["resourceChannel"] = channel_id
+
+        with open("config.json", "w") as f:
+            json.dump(cfg, f)
+
+        self.resource_channel_id = channel_id
+        await ctx.send(f"{rsc_channel.name} has been set as resources channel")
+
+    @commands.command()
+    @commands.has_role("Bot Commander")
+    async def setreviewchannel(self, ctx, channel_id):
+        """Enter ID of channel to set as review channel"""
+        rvw_channel = await self.bot.fetch_channel(channel_id)
+
+        if rvw_channel is None:
+            await ctx.send("Invalid channel id")
+            return
+
+        with open("config.json", "r") as f:
+            cfg = json.load(f)
+            cfg["reviewChannel"] = channel_id
+
+        with open("config.json", "w") as f:
+            json.dump(cfg, f)
+
+        self.review_channel_id = channel_id
+        await ctx.send(f"{rvw_channel.name} has been set as resources channel")
+
+    @commands.command()
+    async def suggestresource(self, ctx, rscname, rsclink, rscdesc, *, rsctags):
+        """Suggest a resource with the name, link, description (in quotes), and tags"""
+        rvw_channel = await self.bot.fetch_channel(self.review_channel_id)
+        message = await rvw_channel.send(
+            embed=create_embed(
+                ctx.message.author.name, rscname, rsclink, rscdesc, rsctags
+            )
+        )
+        await message.add_reaction("✔️")
+        await message.add_reaction("❎")
+        cfg = jsonopen()
```

Comment:
Again, don't use `cfg` as the variable name here and prefer something like `resource_dict` instead